src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,63 @@
+import rightIcon from "../../assets/icon/arrow-right-icon.svg";
+import leftIcon from "../../assets/icon/arrow-left-icon.svg";
+import styles from "./Pagination.module.css";
+import { useEffect, useState } from "react";
+
+const Pagination = ({ currentPage, totalCount, pageSize, onPageChange }) => {
+  const totalPages = Math.ceil(totalCount / pageSize);
+  const groupSize = 5;
+  const currentGroup = Math.ceil(currentPage / groupSize);
+  const startPage = (currentGroup - 1) * groupSize + 1;
+  const endPage = Math.min(startPage + groupSize - 1, totalPages);
+
+  if (totalPages <= 1) return null;
+
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
```

Comment:
💊 제안
이미 button에서 currentPage가 1 일때 disabled 처리를 하셔서 조건문을 빼셔도 될 것 같아요~
```suggestion
  const handlePrevPage = () => {
    onPageChange(currentPage - 1);
  };
```